src/Sylius/Behat/Context/Setup/TaxContext.php

```diff
@@ -146,8 +146,12 @@ private function getAmountFromString($taxRateAmount)
      *
      * @return string
      */
-    private function getCodeFromName($taxRateName)
+    private function getCodeFromNameAndZoneCode($taxRateName, $zoneCode = null)
```

Comment:
Anyway, can we make two methods instead, `getCodeFromName` and `getCodeFromNameAndZoneCode` basing on the previous method?